### PR TITLE
Zy/16 rdflib read remote turtle

### PR DESCRIPTION
**Pull request by zheyxu**
For issue #16

### CODE REVIEW

File: lib/src/term.dart
Comment (gjwgit) [inline lib/src/term.dart:1]: In a file called term.dart I look for a class or function called term?

File: lib/src/graph.dart
Comment (gjwgit) [inline lib/src/graph.dart:1]: This is in desperate need of refactoring - perhaps create a new issue. Again, I expect in this file to find the class Graph. Other functionality into other files?

**Review by gjwgit — CHANGES_REQUESTED** [review]

File: lib/src/graph.dart
Comment (zheyxu) [reply lib/src/graph.dart:1]: May need further discussion.

**Review by gjwgit — APPROVED** [review]

**Review by anushkavidanage — APPROVED** [review]
Functionality works. However, the parser itself does not work with different types of ttl files. We will need separate tasks for that.

**Review by cdawei — APPROVED** [review]

**Pull request merged**